provider/rcode0/rcode0_test.go

```diff
@@ -71,7 +71,7 @@ func TestRcodeZeroProvider_Records(t *testing.T) {
 	if err != nil {
 		t.Errorf("should not fail, %s", err)
 	}
-	require.Equal(t, 6, len(endpoints))
+	require.Equal(t, 10, len(endpoints))
```

Comment:
Can you explain this change?

Reply:
Also "NS" test fixture there https://github.com/kubernetes-sigs/external-dns/blob/ba46a498579165e220bcbfaaba9aa91fd521e5ea/provider/rcode0/rcode0_test.go#L327

After adding NS record support the external-dns will create +4 records: 2 NS and 2 auxiliary TXT ones with the ownership metadata